collectors/health.go

```diff
@@ -152,6 +152,21 @@ type ClusterHealthCollector struct {
 	// This includes object replicas in its count.
 	MisplacedObjectsCount prometheus.Gauge
 
+	// OSDMapFlags
+	OSDMapFlagFull prometheus.Gauge
```

Comment:
Would it be possible to run `gofmt` on this file? That should fix this indentation. Other necessary options are covered here: https://github.com/digitalocean/ceph_exporter/blob/luminous/CONTRIBUTING.md.